### DISCUSSION TITLE
Is Redocusaurus compatible with Redocly's paid services?

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#372] [Q&A] [answered]
Hi Rohit,

I've gone through the issue [#24](https://github.com/rohit-gohri/redocusaurus/issues/24) and understood that you hadn't tried the paid services with Redocusaurus. But it has been a long time since that discussion happened.
Is there a recent update on that front? I might need to use Redocly's paid service to get the "Try-it-out API console."

Any pointers, please.

**Comment by malathikrishnap** [comment]
Hello Rohit,

Any pointers, please? Thanks!

**Reply by rohit-gohri** [reply]
Hey, as far as I know redocly's paid service is a completely different feature - https://redocly.com/docs/api-reference-docs/guides/try-it-console	than the open source package - [Redocly/redoc](https://github.com/Redocly/redoc/).

**Reply by malathikrishnap** [reply]
Thanks for your revert, Rohit.
So, if it is other than Redoc, is it not possible to integrate with Redocusaurus?
I'm afraid, then we will lose the ability to keep the API references within the devdocs portal.

**Reply by malathikrishnap** [reply]
Hi Rohit,
Your input is very important while evaluating the best API Reference tool for our Docusauraus website.
Please let me know whether or not I can use Redocusaurus to integrate Docusaurus with a paid Redocly service.
Thanks!

**Reply by rohit-gohri** [reply, marked as answer]
> Redocusaurus to integrate Docusaurus with a paid Redocly service

Not possible

> Docusaurus with a paid Redocly service

Better to ask the Redocly team, maybe they have something for paid customers

**Comment by malathikrishnap** [comment]
Thank you!
